scripts/update_version.py

```diff
@@ -10,33 +10,11 @@
 
 def update_version():
     """
-    Opens file and updates the version number
+    Updates version numbers and release information across project files
     """
     release_version = os.getenv("VERSION")[1:]
```

Comment:
To make this work, you need to get the VCS version to set this environment variable